### PR TITLE
Setup conda on Readthedocs

### PR DESCRIPTION
**Pull request by zonca**

### CODE REVIEW

**Comment by zonca** [comment]
[healpy fix readthedocs](https://trello.com/c/do7ocHfW/227-healpy-fix-readthedocs)

File: setup.py [hunk 1]
```diff
@@ -76,79 +76,6 @@ def __init__(self, dist):
         build_clib.__init__(self, dist)
         self.build_args = {}
 
-    def autotools_path(self):
```
Comment (lpsinger) [inline setup.py:79]: Eeek! Don't we need this stuff?

Comment (zonca) [reply setup.py:79]: I think it was needed only for readthedocs

Comment (zonca) [reply setup.py:79]: isn't it easier to ask people to install `automake` and `pkg-config` as requirements?

Comment (lpsinger) [reply setup.py:79]: No, this wasn't needed for readthedocs, it was needed for building wheels.

Comment (zonca) [reply setup.py:79]: ok, I'll revert that

Comment (lpsinger) [reply setup.py]: Actually, never mind, I think this is OK. I misread the diff.

**Pull request merged**